### PR TITLE
[Docs] Notification usage docs

### PR DESCRIPTION
**Pull request by quaark**
Added documentation for using the notifications SDK features.

### CODE REVIEW

**Review by liranbg — APPROVED** [review]

File: docs/runtimes/notifications.md [hunk 1]
```diff
@@ -0,0 +1,131 @@
...
+## Setting Notifications on Live Runs
+You can set notifications on live runs via the `set_run_notifications` method. For example:
+
+```python
+import mlrun
+
+mlrun.get_run_db().set_run_notifications("<project-name>", "<run-uid>", [notification1, notification2])
+```
+
+Note that this overrides any existing notifications on the run. So deleting all notifications is done by passing an empty list.
```
Comment (jillnogold) [inline docs/runtimes/notifications.md:98]: Note that this overrides any existing notifications on the run. To delete all notifications, pass an empty list.

File: docs/runtimes/notifications.md [hunk 2]
```diff
@@ -0,0 +1,131 @@
+(notifications)=
+
+# Notifications
+
+MLRun supports configuring notifications on jobs and scheduled jobs. In this section, 
+the SDK for using notifications will be explained in detail.
```
Comment (jillnogold) [inline docs/runtimes/notifications.md:6]: This section describes the SDK for notifications.

File: docs/runtimes/notifications.md [hunk 3]
```diff
@@ -0,0 +1,131 @@
...
+- [Notification Kinds](#notification-kinds)
+- [Configuring Notifications For Runs](#configuring-notifications-for-runs)
+- [Configuring Notifications For Pipelines](#configuring-notifications-for-pipelines)
+- [Setting Notifications on Live Runs](#setting-notifications-on-live-runs)
+- [Setting Notifications on Scheduled Runs](#setting-notifications-on-scheduled-runs)
+- [Notification Conditions](#notification-conditions)
+
+
+## The Notification Object
+The notification object's schema is as follows:
```
Comment (jillnogold) [inline docs/runtimes/notifications.md:20]: The notification object's schema is:

File: docs/runtimes/notifications.md [hunk 4]
```diff
@@ -0,0 +1,131 @@
...
+- `kind`: str - notification kind (slack, git, etc...)
+- `when`: list[str] - run states on which to send the notification (completed, error, aborted)
+- `name`: str - notification name
+- `message`: str - notification message
+- `severity`: str - notification severity (info, warning, error, debug)
+- `params`: dict - notification parameters (See definitions in [Notification Kinds](#notification-kinds))
+
+
+## Local vs Remote
+Notifications can either be sent locally from the SDK, or remotely from the MLRun API.
```
Comment (jillnogold) [inline docs/runtimes/notifications.md:30]: Notifications can be sent either locally from the SDK, or remotely from the MLRun API. Usually, a local run sends locally, and a remote run sends remotely.

**Pull request merged**